onyo/lib/onyo.py

```diff
@@ -624,15 +622,18 @@ def _fsck_pseudo_keys(self) -> bool:
         pseudo_keys = ["type", "make", "model", "serial"]
 
         for asset in self.assets:
-            violation_list = [x for x in pseudo_keys if x in self._read_asset(asset)]
+            violation_list = [
+                x for x in pseudo_keys if x in self._read_asset(asset)]
             if violation_list:
                 assets_failed[asset] = violation_list
 
         if assets_failed:
-            log.error(f"Pseudo keys ({', '.join(pseudo_keys)}) are reserved for asset names, and are not allowed in asset files. The following assets contain pseudo keys:\n" +
-                      self._n_join([f'{asset}: ' +
-                                    ', '.join([k for k in assets_failed[asset]])
-                                    for asset in assets_failed]))
+            log.error(
+                'Pseudo keys {0} are reserved for asset names, and are not '
+                'allowed in asset files. The following assets contain pseudo '
+                'keys:\n{1}'.format(tuple(pseudo_keys), '\n'.join(
```

Comment:
IMO, this is more readable if the second item passed to .`format()` is on the next line.

Purely a matter of taste. Feel free to disregard if you disagree.

Reply:
I have no problem with this, and I agree that it looks better. Changed.